.github/workflows/ci.yml

```diff
@@ -25,13 +25,30 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - python: 3.6
+          - framework: tensorflow
```

Comment:
```suggestion
          - framework: tensorflow1
```

Reply:
I'm wondering whether there is any way of naming each matrix element so that we can distinguish them in the run names as` Run tensorflow1 Tests`(Instead of ` Run legacy Tests`) and `Run tensorflow2 Tests`? Both matrix elements should have the framework variable set to `tensorflow`.